src/components/PageSections/HowYouKnowMenu/_HowYouKnowMenu.scss

```diff
@@ -3,7 +3,8 @@
   display: none;
   position: fixed;
   top: 0;
-  width: 100vw;
+  width: 100%;
+  max-width: $coa-top-menu-max-width;
```

Comment:
so here were setting max width with the later defined variable as part of this component. which is cool. 
for the header above we add a container modifier that does this + centers with the auto margins?